src/scripts/populateDatabase.ts

```diff
@@ -0,0 +1,10 @@
+import { populateAgendaItems } from '@/database/pipelines/populateAgendaItems';
+
+const beginningOfTerm = 883630800000;
```

Comment:
I assume this works, but is definitely the kinda thing that makes me nervous about timezones potentially messing with.
Maybe in a future pass we could use a DateStruct instead of the built in Date obj just to avoid potential headaches